src/msw/renderer.cpp

```diff
@@ -1011,7 +1012,10 @@ wxRendererXP::DrawItemSelectionRect(wxWindow *win,
                                     const wxRect& rect,
                                     int flags)
 {
-    wxUxThemeHandle hTheme(win, L"EXPLORER::LISTVIEW");
+    wxScopedPtr<wxUxThemeHandle> hThemePtr(new wxUxThemeHandle(win, L"EXPLORER::LISTVIEW"));
```

Comment:
According to the [docs](https://docs.microsoft.com/en-us/windows/win32/api/uxtheme/nf-uxtheme-openthemedata#remarks), it should be possible to pass multiple class names, i.e., something like this
`wxScopedPtr<wxUxThemeHandle> hThemePtr(new wxUxThemeHandle(win, L"EXPLORER::LISTVIEW;LISTVIEW"));`

This could make the code simpler but I did not try if it works as expected here.

Reply:
Thanks, I didn't know this. I'll try it later and update the PR if it works.

Reply:
I tried this before and it then works (only in this order, not reversed) for items in both the dataview and render sample. Tested with Windows 10 only, but support for passing multiple classes has been around since incarnation AFAIK. There's no need for the scope changes, merely changing the two strings is enough.